ipatests/test_xmlrpc/test_location_plugin.py

```diff
@@ -126,11 +127,12 @@ def test_delete_location(self, location):
     not api.Command.dns_is_enabled()['result'], reason='DNS not configured')
 class TestLocationsServer(XMLRPC_test):
     messages = [{
-        u'data': {u'service': u'named-pkcs11.service',
+        u'data': {u'service': u'%s' % knownservices.named.systemd_name,
```

Comment:
The ``%`` format operation is not necessary here:
``u'service': knownservices.named.systemd_name,``

Reply:
Ah, yes, it is.
Pushed.